lib/literal/enum.rb

```diff
@@ -199,7 +199,14 @@ def name
 	end
 
 	alias_method :inspect, :name
-	alias_method :to_s, :name
+
+	def to_s
+		self.class.names[self].name
+	end
+
+	def to_sym
+		self.class.names[self]
+	end
```

Comment:
```suggestion
	def to_sym
		self.class.names[self]
	end

	def to_s
		to_sym.to_s
	end
```

IMO ^^ will be a bit better

Reply:
I don't think we need to_s to return a mutable string. You can always get that by duping the frozen string.

Reply:
Oh. That's a good point. I'm totally fine with:

``` ruby
def to_s
  to_sym.name
end
```

That way we will avoid useless object allocations indeed